astropy/utils/console.py

```diff
@@ -498,6 +498,19 @@ def human_file_size(size):
     return "{0:>3s}{1}".format(str_value, suffix)
 
 
+class _mapfunc(object):
```

Comment:
I'm not sure this warrants (the overhead of) a class. 

Such operations are typically written as "decorators":

```
def _mapfuncfunc(func):
    def inner(i_arg):
        i, arg = i_arg
        return i, func(arg)
    return inner
```